content/en/core/releases/4.11.0.md

```diff
@@ -0,0 +1,106 @@
+
+---
+title: "4.11.0 release notes"
+linkTitle: "4.11.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.11.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+None.
+
+## Highlights
+
+### The CHT-Datasource API now supports querying persons
+
+The CHT datasource API has been expanded to allow querying places and persons by contact type.
+
+##### Details
+See more details in the [api docs](http://docs.communityhealthtoolkit.org/cht-datasource/).
+
+### Phone numbers are not required to be checked for duplicates
+Phone number fields can now be used without automatically checking for duplicates. This can be useful for workflows where users may share a phone or for any workflow where having duplicate phones is possible, and it's undesirable to check for duplicates.
+
+Duplicate checking is still available if desired.
+
+##### Details
+- Previously, the phone widget was configured in an xlsxform field by setting `type: tel`. By default, the phone widget both validates the phone number value and checks for existing contacts that have the same number.
+- With these new changes, the existing (now deprecated) functionality stays the same, but going forward, phone number fields should be configured in an xlsxform by setting `type: string` with `appearance: number tel`.
+- Fields with this new configuration will always validate the phone number value, but by default they will not check for existing contacts with the same number.
+- To enforce a unique phone number with the new style of field, you should add a new column with `instance::cht:unique_tel: true`.
+
+### The logged in users contact summary contains more target documents
```

Comment:
I think the highlight should be something like: 
`Last three target documents passed to contact-summary function`. 
Right now it implies that this only applies to the logged in user's contact, which is not true.